zktest/test/java/org/zkoss/zktest/zats/test2/F85_ZK_3913Test.java

```diff
@@ -0,0 +1,46 @@
+/* F85_ZK_3913Test.java
+
+        Purpose:
+                
+        Description:
+                
+        History:
+                Tue Apr 24 4:22 PM:55 CST 2018, Created by klyve
+
+Copyright (C) 2018 Potix Corporation. All Rights Reserved.
+*/
+package org.zkoss.zktest.zats.test2;
+
+import org.junit.Test;
+import org.zkoss.zats.mimic.ComponentAgent;
+import org.zkoss.zktest.zats.WebDriverTestCase;
+import org.zkoss.zktest.zats.ztl.JQuery;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public class F85_ZK_3913Test extends WebDriverTestCase {
+
+	@Test
+	public void test() {
+		connect();
+		JQuery canvas = jq("canvas");
+		assertEquals(3, canvas.length());
+		
+		String height = getEval("jq(\"canvas\")[0].height");
+		waitResponse();
+		String width = getEval("jq(\"canvas\")[0].width");
+		waitResponse();
+		assertEquals("100", height);
+		assertEquals("114", width);
```

Comment:
I don't think checking the height/width is a good idea.

Checking if something being drawn may be better: https://stackoverflow.com/q/4644524